### ISSUE TITLE
Save and Apply partial visible until reaching the bottom

### ISSUE THREAD

**Issue by rtgibbons** [djonsson/jenkins-atlassian-theme#57]
The Save and Apply buttons are only partially visible until we've scrolled all the way to the bottom.

![screenshot 2015-01-23 10 07 25](https://cloud.githubusercontent.com/assets/8000/5877855/379fab52-a2e8-11e4-8f01-134c2696d198.png)

I took a look at how the un-styled Jenkins works, and they are completely hidden until you've reach the bottom of the page; and then they are shown. The code for this seems to be in hudson-behavior.js around line 1157 (Using Safari Inspector).

But the comment states

````
/*
    Use on div tag to make it sticky visible on the bottom of the page.
    When page scrolls it remains in the bottom of the page
    Convenient on "OK" button and etc for a long form page
*/
````

So not sure if this behavior was changed on purpose, or if it's a bug in their code

This is on Jenkins 1.597.

**Comment by ctrl-f5** [comment]
this makes some pages unusable, also the console log is not visible to the bottom.

The wole page body is "position: fixed", there must be a better solution to this...

**Comment by rtgibbons** [comment]
@ctrl-f5 Yeah - but this `position: fixed` is coming from Jenkins; so we are either stuck working with it or submitting a larger changes to Jenkins

**Comment by ctrl-f5** [comment]
aha, I see, I'll try to dig into it this weekend. Either way, it's always possible to override it, depending on how much jenkins relies on it.

**Comment by rtgibbons** [comment]
This looks to be a bug in Jenkins - should be fixed on the next release (1.598)

https://issues.jenkins-ci.org/browse/JENKINS-26312

**Comment by rtgibbons** [comment]
This was fixed upstream in 1.597
